### PR TITLE
Jetpack 10.8-a.3 Atomic Release

### PR DESCRIPTION
**Pull request by samiff**
#### Changes proposed in this Pull Request:

Regularly scheduled 10.8-a.3 Atomic-only Jetpack release.

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

Proofread

### CODE REVIEW

File: projects/packages/my-jetpack/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.8] - 2022-03-08
+### Added
+- Added connected plugins slugs to My Jetpack tracking events
+- Add link to jetpack.com in the footer
```
Comment (sdixon194) [inline projects/packages/my-jetpack/CHANGELOG.md:11]: I believe this is for My Jetpack as well. 

Could probably just do one general "My Jetpack: Various design related improvements"  or something and consolidate a few other entries as well.

Comment (samiff) [reply projects/packages/my-jetpack/CHANGELOG.md:11]: Looks like that is the my-jetpack changelog

File: projects/plugins/jetpack/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,38 @@
...
+
+### Improved compatibility
+- Premium Content Block: prevent block from being nested inside itself.
+- Various Blocks: remove deprecated attributes from Button components.
+- CLI Tools: ensure WP CLI is present before extending the class.
+
+### Bug fixes
+- Jetpack: remove the duplicated `jetpack_relatedposts_returned_results filter`.
+- Premium Content Block: login button now redirects to the correct post instead of to a 404 page when the URL contains characters that have been previously encoded.
+- Payment Blocks: fixed and issue with the upgrade banner being obscured from all payment blocks.
```
Comment (sdixon194) [inline projects/plugins/jetpack/CHANGELOG.md:22]:
```suggestion
- Payment Blocks: fixes an issue with the upgrade banner being obscured from all payment blocks.
```

**Review by sdixon194 — APPROVED** [review]

**Review by coder-karen — APPROVED** [review]

**Comment by samiff** [comment]
r241316-wpcom

**Pull request merged**